### ISSUE TITLE
Facebook debugger can't find image

### ISSUE THREAD

**Issue by gorgekara** [tomitrescak/meteor-uploads#236]
I've been having some issues with the uploaded images with your package. I followed your installation instructions and everything works ok but for some reason the facebook link debugger thinks the image size is over 8MB (it's around 105Kb).

Here is the error output i get:
```
Provided og:image URL, http://snapbin.net/upload/GjRR6oR3ybqw_1465040165800.jpg could not be downloaded because it exceeded the maximum allowed sized of 8Mb.
```

I've setup meteor SEO and pretty sure i'm serving the correct link to the image in the correct meta tag.

Think it has something to do with the redirects but i'm not sure - tried moving the upload folder from ./upload to /public/upload but then the page is refreshed every time an image is uploaded (bit weird).

**Comment by tomitrescak** [comment]
Wow, I do not know even where to start ;( Never bumped into this. Try more broad Meteor community.

**Comment by gorgekara** [comment]
Yeah, I thought it was really weird too - tried gitter and stackoverflow.
But since I needed a quick fix I switched up your server and make my nodejs
image server.

Will try again with yours though have to find out why it was behaving like
that.

Thanks

On Tue, Jun 7, 2016 at 9:55 AM, Tomas Trescak <notifications@github.com>
wrote:

> Wow, I do not know even where to start ;( Never bumped into this. Try more
> broad Meteor community.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tomitrescak/meteor-uploads/issues/236#issuecomment-224206921>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABOonOcDeGfZX0fVG7MSnFzzQ0Eomuveks5qJSPggaJpZM4IuLOr>
> .
>



-- 
Gjorge Karakabakov
Mob.: +38977551014
Twitter <http://www.twitter.com/dark4p> Google+
<https://plus.google.com/104370264995488150366> LinkedIn
<http://www.linkedin.com/pub/gjorge-karakabakov/3a/b17/442>

**Comment by tomitrescak** [comment]
My guess are the headers. Check which headers I'm setting and which headers needs Facebook.
